app/components/add-validations-to-form.js

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { GRAPHS } from '../controllers/formbuilder/edit';
+import { ForkingStore } from '@lblod/ember-submission-form-fields';
+import { getLocalFileContentAsText } from '../utils/get-local-file-content';
+import { FORM, RDF } from '../utils/rdflib';
+import { areValidationsInGraphValidated } from '../utils/validation-shape-validators';
+
+export default class AddValidationsToFormComponent extends Component {
+  @tracked builderStore;
+  @tracked builderForm;
+  @tracked formTtlCode;
+  @tracked showRdfForm = false;
+  @tracked sourceNode;
+
+  graphs = GRAPHS;
+  form_ttl_path = '/forms/validation/form.ttl';
+  meta_ttl_path = '/forms/validation/meta.ttl';
```

Comment:
nitpick: What's up with the snake_case naming? 😄

Reply:
It's just to make it clear that these are not variables that.. yeah I can change it so just camelCasing :sweat_smile:

Reply:
Yea, in that case I would make them const variables in the module scope, or simply use the strings in the function directly 😄.